pkg/addon-operator/converge.go

```diff
@@ -27,9 +26,18 @@ const (
 	WaitAfterAll            ConvergePhase = "WaitAfterAll"
 )
 
+type FirstConvergePhase string
```

Comment:
We use it only for tracking an inner state. Don't you think we can use `int` here and `= iota` for initialization.
Comparing integers would be faster until we don't need string representation

Reply:
Done